### ISSUE TITLE
SABnzdb history tab not working

### ISSUE THREAD

**Issue by danielstorch** [CometTools/LunaSea#583]
The history tab for SABnzb is not working for me at all. It always fails loading and it is always the same error.

```
    {
        "timestamp": "2021-12-27T00:48:23.971",
        "type": "Error",
        "class_name": "package:lunasea/modules/sabnzbd/core/api/api.dart",
        "method_name": "SABnzbdAPI.getHistory",
        "message": "SABnzbd: Failed to fetch history",
        "error": "type '_OneByteString' is not a subtype of type 'List<dynamic>'",
        "stack_trace": [
            "package:lunasea/modules/sabnzbd/core/api/api.dart 347  SABnzbdAPI.getHistory",
            "package:flutter/src/widgets/async.dart                 _FutureBuilderState._subscribe.<fn>"
        ]
    },
 ```

Anythin I can do to fix this or is it a bug in the app?

This is on iOS

**Comment by JagandeepBrar** [comment]
Hey @danielstorch,

You're actually experiencing the same problem as here: https://github.com/CometTools/LunaSea/issues/289

TL;DR: A broken history record is trying to be parsed, take a look at your SABnzbd history via the web and look for any failed entries in your history and remove them.

**Comment by danielstorch** [comment]
Deleting failed entries fixed it, thanks 👍
